src/main/resources/application.yaml

```diff
@@ -1,7 +1,8 @@
 spring:
   datasource:
     driver-class-name: com.mysql.cj.jdbc.Driver
-    url: jdbc:mysql://localhost:3307/warehouse
+#    url: jdbc:mysql://localhost:3306/warehouse # for local startup
```

Comment:
The rule: don't commit commented code.

There is only one exception: `// TODO` comment:
```kotlin
// TODO [<ticket-number>]: <do-something>, <when>, etc 
```